deploy/crds/hypercloud.tmaxanc.com_v1alpha1_virtualmachineimage_local_cr.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: hypercloud.tmaxanc.com/v1alpha1
+kind: VirtualMachineImage
+metadata:
+  name: localvmim
+spec:
+  source:
```

Comment:
how the vimv works when both http and volume has no value or with value?

Reply:
If neither http nor volume is exist, vmim is error.
If both http and voluem is exist, vmim will use http.